src/Lumi/Styles/Responsive.purs

```diff
@@ -0,0 +1,19 @@
+module Lumi.Styles.Responsive where
+
+import Prelude
+
+import Lumi.Components (PropsModifier)
+import Lumi.Styles (Style, css, nested, styleModifier)
+
+desktopQuery :: Style -> Style
+desktopQuery style =
+  css
+    { "@media (min-width: 860px)": nested style
+    }
+
+onDesktop :: forall props. PropsModifier () -> PropsModifier props
```

Comment:
Should `props` be passed to both of these types, or `props1` and `props2`? The empty row seems restrictive.

Reply:
We can't have an open row here, otherwise we wouldn't be able to apply the argument to the props set, because we don't know what it is. It's not restrictive given that this should only work for styles anyway, and style modifiers are always `forall props. PropsModifier props`.